sysutils/resticprofile/Portfile

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/creativeprojects/resticprofile 0.27.1 v
+go.offline_build    no
+github.tarball_from archive
+revision            0
+
+homepage            https://creativeprojects.github.io/resticprofile/
+
+description         Configuration profiles manager and scheduler for restic
+
+long_description    resticprofile is one of many automation tools for restic, \
+                    also called a wrapper. In a nutshell, resticprofile \
+                    provides a configuration file and a runner that will \
+                    generate all the necessary calls to restic. \
+                    Additionally, it can schedule a profile via a user agent or \
+                    a daemon in launchd
+
+categories          sysutils
+installs_libs       no
+license             GPL-3
+maintainers         {fsoj.de:lheise @lucaheise} \
+                    openmaintainer
+
+checksums           rmd160  b4a704c4d88e6f3f795d62768a16ea325eebf726 \
+                    sha256  9c9b5b5e56c633ab4969355fff139b3c36979f88b815bdd02ff8d1dd14e69765 \
+                    size    3195445
+
+depends_run         port:restic
+
+destroot {
+    xinstall -m 0755 ${worksrcpath}/${name} ${destroot}${prefix}/bin/
```

Comment:
Is there no documentation to install?

Reply:
there are no man-pages.
The documentation HTML could be built with Hugo from the docs folder in the repo. It doesn't look like the project assumes that someone will build it themselves, as the URL paths are built in. 

As in the alternative autorestic port, I have left this out for now. I couldn't use the restic port as example, as it uses a different HTML generator.

I could try it, but I haven't found an example of a port in a similar situation. I don't see my self able to try to implement this as of now.